docs/reference/data-streams/change-mappings-and-settings.asciidoc

```diff
@@ -0,0 +1,520 @@
+[[data-streams-change-mappings-and-settings]]
+== Change mappings and settings of backing indices
+++++
+<titleabbrev>Change mappings and settings</titleabbrev>
+++++
+
+Each data stream has an <<create-a-data-stream-template,associated composable
+template>>. Mappings and index settings from this template are applied to new
+backing indices created for the stream. This includes the stream's first
+backing index, which is auto-generated when the stream is created.
+
+Before creating a data stream, we recommend you carefully consider which
+mappings and settings to include in this template. However, if you later need to
+change the mappings or settings of a data stream's backing indices, you have a
+couple options:
+
+* To apply changes to future backing indices, simply update the composable
+template associated with the data stream. Mapping and setting changes will be
+automatically applied to any backing indices created after the update.
```

Comment:
Just a heads-up: this is currently accurate though we're considering applying mappings at the level of the data stream instead of for each backing index.

Reply:
Thanks for noting that. I'll leave as-is for now, but I think we could easily remove the mapping references later on.